src/i18n/fra.json

```diff
@@ -0,0 +1,88 @@
+{
+  "locale": "fra",
```

Comment:
Something I'm still a little fuzzy on is how all the moment locale stuff works (as you may have seen from previous PRs/reviews haha).  Is that something you looked into at all? It seems there is a `fr`  locale available so should that be used here?

Reply:
Well I thought I understood it till I was double checking it. I was just putting in the 3 letter code from the linked wikipedia article in the Language Translation document, and the places I was seeing relative time things (e.g. "last updated 3 days ago") was translating, so I thought I did that correctly. But you're right, moment.js lists the locale code as fr, not fra. And when I double checked that just now, it's leaving it translated in the wrong language, even when I change the locale to "fr". So I'll need to poke around more. Good catch!

Reply:
ah ha! So moment can let you download all their default locales at once using the standard npm install moment, but that takes up a lot of space, so in #111 , it was updated so we'd only pull in the default momentjs locales that we were currently using, and our custom locales. 
We can get copies of the momentjs locales on their github: https://github.com/moment/moment/tree/develop/locale, and copy them int the web_modules/moment/dist folder, then load them into the index.js file. Once I did that, the moment.js translations for french work (also after I updated the fra.json locale to "fr"). 
I'll do a new push with those changes and updating the translation doc with this info now.

Reply:
Fantastic! Great find! I think this was one of the last pieces of moment that wasn't quite clicking in our set up so thanks for digging into that!